rednotebook/gui/clouds.py

```diff
@@ -137,21 +141,16 @@ def _get_cloud_body(self, cloud_words):
             delta_count = 1
 
         min_font_size = 10
-        max_font_size = 40
```

Comment:
I think it's good to tie 40 to the `max_font_size` name, so please revert to the previous code.